documentation/src/main/markdown/releasenotes.md

```diff
@@ -10,6 +10,7 @@
 ### Changed features
 
 * Evicted dependencies in Simple Build Tool(SBT) projects will no longer be included in the Bill of Materials(BoM) generated during the scan.
+* The maximum polling interval threshold is now dynamic when [solution_name] polls Black Duck for the Bill of Materials(BoM) status and is dependent upon, and optimized for, the specific scan size. (The maximum polling threshold was formerly a fixed 60-second value.)
```

Comment:
I believe this change also applies to the waiting for stateless scans? We should probably mention that too?

Reply:
@dterrysynopsys Was this mechanism not applied to Stateless scans in the prior release already?

Reply:
I believe the general Fibonacci waiting was applied but I want to say that Nirav's latest changes update that as well as the bom status waiting. @niravrsynopsys for confirmation.

Reply:
So the waiting for scan results in fibonacci sequence applies to Intelligent mode only when `detect.wait.for.results=true`.  For rapid/stateless mode, we always poll in a fibonacci sequence irrespective of the value provided in `detect.wait.for.results`. This latest change sets the max threshold for both of these cases (rapid and intelligent).
The question I have is - should we mention these conditional details in our release note or is saying "when [solution_name] polls Black Duck for the Bill of Materials(BoM) status" sufficient?

Reply:
@sig-cpotts Please suggest based on the above conversation.

Reply:
Something like this if we want to cover our bases?

* The maximum polling interval threshold is now dynamic when [solution_name] polls Black Duck for results. This dynamic threshold is dependent upon, and optimized for, the specific scan size. (The maximum polling threshold was formerly a fixed 60-second value.)

Reply:
Updated. Thanks!